src/BloomExe/UpdateVersionTable.cs

```diff
@@ -88,8 +88,12 @@ public UpdateTableLookupResult LookupURLOfUpdate()
 					continue; //comment
 
 				var parts = line.Split(new[] {','}, StringSplitOptions.RemoveEmptyEntries);
-				if(parts.Length!=3)
-					throw new ApplicationException("Could not parse a line of the UpdateVersionTable on "+URLOfTable+" '"+line+"'");
+				if (parts.Length != 3)
+				{
+					Logger.WriteEvent("***Error: UpdateVersionTable could not parse line {0} of this updateTableContent:", line);
+					Logger.WriteEvent(TextContentsOfTable);
```

Comment:
Surprised you don't want to log the URL where we are finding the particular update version table (which was part of the original message)